File: Base/src/main/java/picture/luck/picture/lib/basic/IBridgeLoaderFactory.java
package picture.luck.picture.lib.basic;

import picture.luck.picture.lib.loader.IBridgeMediaLoader;

/**
 * @author：luck
 * @date：2022/6/10 9:37 上午
 * @describe：IBridgeLoaderFactory
 */
public interface IBridgeLoaderFactory {
    /**
     * CreateLoader
     */
    IBridgeMediaLoader onCreateLoader();
}

File: PictureSelector/src/main/java/com/luck/picture/lib/basic/IBridgeLoaderFactory.kt
package com.luck.picture.lib.basic

import com.luck.picture.lib.loader.IBridgeMediaLoader

/**
 * @author：luck
 * @date：2022/6/10 9:37 上午
 * @describe：IBridgeLoaderFactory
 */
interface IBridgeLoaderFactory {
    /**
     * CreateLoader
     */
    fun onCreateLoader(): IBridgeMediaLoader?
}